### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by santitigaga**

### CODE REVIEW

File: async/freemarker-converter-async/src/main/java/co/com/bancolombia/FreeMarkerConverseAsync.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Mono;
+
+@Service
+@AllArgsConstructor
+public class FreeMarkerConverseAsync implements ConverseDataGatewayAsync {
+
+    private final FreeMarkerConverseSync freeMarkerConverseSync;
+
+    @SneakyThrows
```
Comment (juancgalvis) [inline async/freemarker-converter-async/src/main/java/co/com/bancolombia/FreeMarkerConverseAsync.java]: Please remove this unused annotations

File: main.gradle [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        }
+    }
+    compileJava {
+        options.compilerArgs = [
+                '-Amapstruct.suppressGeneratorTimestamp=true',
+                '-Amapstruct.suppressGeneratorVersionInfoComment=true'
+        ]
+    }
+
+//    if (toPublish.split(',').contains(it.name)) {
```
Comment (juancgalvis) [inline main.gradle]: Remove this sentence

**Review by juancgalvis — CHANGES_REQUESTED** [review]

File: async/freemarker-converter-async/src/main/java/co/com/bancolombia/FreeMarkerConverseAsync.java (hunk 1 above)
Comment (santitigaga) [reply async/freemarker-converter-async/src/main/java/co/com/bancolombia/FreeMarkerConverseAsync.java]: Done

File: main.gradle (hunk 2 above)
Comment (santitigaga) [reply main.gradle]: Removed

**Review by juancgalvis — APPROVED** [review]
That's great

**Pull request merged**